example/src/screens/Examples/AwareScrollViewStickyFooter/StickyFooter.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { memo, useMemo, type ReactNode } from 'react';
+import { ViewProps, type ViewStyle } from 'react-native';
+import { useReanimatedKeyboardAnimation } from 'react-native-keyboard-controller';
+import Animated, {
+  interpolate,
+  useAnimatedStyle,
+} from 'react-native-reanimated';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+export interface StickyFooterProps extends Pick<ViewProps, 'onLayout'> {
+  children: ReactNode;
+  /**
+   * The offset to add when the keyboard is closed. Useful to position the
+   * sticky bar above things such as a tab bar.
+   */
+  bottomOffsetWhenKeyboardIsClosed?: number;
+}
+
+export const StickyFooter = memo(
+  ({
+    children,
+    bottomOffsetWhenKeyboardIsClosed,
+    ...props
+  }: StickyFooterProps) => {
+    const { height, progress } = useReanimatedKeyboardAnimation();
+    const { bottom } = useSafeAreaInsets();
+
+    const actionBarAnimatedStyles = useAnimatedStyle(() => {
+      const translateY = interpolate(progress.value, [0, 1], [0, height.value]);
```

Comment:
I think on iOS it'll be perfectly synchronised, but on Android it will have kind of parallax effect?

On iOS when keyboard will appear `useReanimatedKeyboardAnimation` will immediately switch `height` to `336` (or any other number when keyboard is opened). On Android hook works slightly differently and it'll return values something like (values are taken from my head, just for explaining a problem):
- 0.1 (progress), 20 (height) -> 2px (interpolated), expected is 20px
- 0.2 (progress), 40 (height) -> 8px (interpolated), expected is 40px
- ...
- 1 (progress), 336 (height) -> 336px (interpolated and expected)

Can't we just use `height.value` as `translateY` here?

Reply:
Also it seems like we interpolate through entire keyboard height from `0` to `keyboardHeight`. But I'm just thinking, that in some cases people may treat something as `StickyFooter`, though this view can be not sticky to the bottom trait of the screen (for example various pencil icons in left bottom corner that are not sticky to any of screen border).

Maybe we should take an approach from `KeyboardAvoidingView`? In [onLayout](https://github.com/kirillzyusko/react-native-keyboard-controller/blob/main/src/components/KeyboardAvoidingView/index.tsx#L82) we are memoizing current layout (if keyboard is closed) and calculate interpolation based on screen dimensions, keyboard position and memoized layout?

I haven't tested your implementation yet and maybe `bottomOffsetWhenKeyboardIsClosed` does this thing, but in my opinion it would be better to keep API of a component as simple as possible. Let me know what do yu think about it 👀